### DISCUSSION TITLE
UI glitch with branch dropdown

### DISCUSSION THREAD

**Discussion by herbievine** [community/community#114290] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

When viewing a repo, if you click the dropdown and start scrolling, the dropdown follows...

Chrome 122.0.6261.128 (Official Build) (arm64)
Mac OS Sonoma

https://github.com/community/community/assets/41120535/ae632397-47b2-4987-b8a0-ad95beae8daf

**Comment by look** [comment]
Thanks for the report, I can also reproduce the issue. I've let the team know.
